### PR TITLE
ddl: Add a safeguard for lease not found error

### PR DESCRIPTION
**Pull request by zimulala**
If the etcd server turns clocks forward，the following case may occur.
The etcd server deletes this session's lease ID, but etcd session doesn't find it.
In this time if we do the campaign operation, the etcd server will return ErrLeaseNotFound.

### CODE REVIEW

File: ddl/owner_manager.go [hunk 1]
```diff
@@ -191,6 +193,14 @@ func (m *ownerManager) campaignLoop(ctx goctx.Context, etcdSession *concurrency.
 			log.Infof("[ddl] %s break campaign loop err %v", idInfo, err)
 			return
 		default:
+			// If the etcd server turns clocks forward，the following case may occur.
+			// The etcd server deletes this session's lease ID, but etcd session doesn't find it.
+			// In this time if we do the campaign operation, the etcd server will return ErrLeaseNotFound.
+			if terror.ErrorEqual(err, rpctypes.ErrLeaseNotFound) {
```
Comment (shenli) [inline ddl/owner_manager.go]: When will the `err` variable be set?

Comment (zimulala) [inline ddl/owner_manager.go]: This kind of error will be set at line 207.

Comment (winkyao) [inline ddl/owner_manager.go]: why not check the err in line 208?

**Comment by zimulala** [comment]
PTAL @shenli @winkyao

**Comment by shenli** [comment]
@disksing PTAL

**Comment by shenli** [comment]
LGTM

**Comment by disksing** [comment]
LGTM

**Pull request merged**